packages/xrpl/src/client/index.ts

```diff
@@ -125,9 +125,24 @@ import {
 } from './partialPayment'
 
 export interface ClientOptions extends ConnectionUserOptions {
+  /**
+   * Factor to multiply estimated fee by to provide a cushion in case the
```

Comment:
```suggestion
   * Multiplication factor of the estimated fee to provide a buffer, in case the
```

Reply:
Used 1st half of change but kept the `cushion` since this parameter is called `feeCushion` and `buffer` is a meaningful data type name we don't want to accidentally reference :)